functions/create_reminder.ts

```diff
@@ -51,16 +50,45 @@ export default SlackFunction(
       return { error: `Failed to create reminder: ${putResponse.error}` };
     }
 
-    // Search for any existing triggers for the welcome workflow
-    const triggers = await findMeetingReminderTrigger(client, channel);
-    if (triggers.error) {
-      return { error: `Failed to lookup existing triggers: ${triggers.error}` };
+    const setupResponse = await setupMeetingReminder(
+      client,
+      channel,
+      date,
+      message,
+    );
+    if (setupResponse.error) {
+      return { error: `Failed to setup reminder: ${setupResponse.error}` };
     }
+    // Search for any existing triggers for the welcome workflow
+    // const triggers = await findMeetingReminderTrigger(client, channel);
+    // if (triggers.error) {
+    //   return { error: `Failed to lookup existing triggers: ${triggers.error}` };
+    // }
 
     return { outputs: {} };
   },
 );
 
+export function setupMeetingReminder(
+  client: SlackAPIClient,
+  channel: string,
+  date: number,
+  message: string,
+): Promise<{ ok: boolean; error?: string }> {
+  try {
+    const result = client.chat.scheduleMessage({
+      channel: channel,
+      text: message,
+      post_at: date.toString(),
```

Comment:
Did we have a reason use toString() here? Based on the docs, it looks like it's expecting a number: https://api.slack.com/methods/chat.scheduleMessage